ui/app/styles/core/buttons.scss

```diff
@@ -327,3 +327,17 @@ a.button.disabled {
   font-weight: inherit;
   cursor: pointer;
 }
+
+// Modification to HDS copy button component to match current UI
+.hds-copy-button {
+  &.icon-only {
+    background: none;
+    border: none;
+    box-shadow: none;
+    min-width: $spacing-l;
+    padding: 0;
+  }
+  &.white-icon > * {
```

Comment:
we try to avoid universal selection `*` - was this the only way to get the icon to be white?

Reply:
Thanks for letting me know. Changing this to .white-icon svg {...} to target the icon; it appears this pattern is used elsewhere in the code